docs/conf/australia/2018/schedule.rst

```diff
@@ -46,43 +52,56 @@ The unconference sessions run in parallel with the workshop.
 * **When**: **14:30-16:30**
 * **Details**: :doc:`/conf/{{shortcode}}/{{year}}/unconference`
 
-Reception
-~~~~~~~~~
+Thursday Night Social
+~~~~~~~~~~~~~~~~~~~~~~~
 
-We encourage everyone to drop by on Thursday evening for the conference reception.
-You'll have a chance to get acquainted with each other over some drinks and snacks.
-We'll also help groups organize dinner plans, so you can continue your conversations over more substantial food as well.
+The official Write the Docs Australia social!
+This event is for **conference attendees only**, so please bring your badge to be let into the venue.
+There will be light snacks and drinks available on the conference while our tab lasts.
 
 * **Where**: TBA
-* **When**: **17:30-21:00**
+* **When**: **5.30pm-8pm**
 
 
 Friday, 16th November 2018
 ----------------------------------------
 
-*Tea and coffee will be provided, as well as snacks and drinks all day.*
+*Tea, coffee and lunch will be provided, as well as snacks and drinks all day.*
 
 .. contents::
```

Comment:
I think we should remove the table of contents on this page. We don't have a corresponding table of contents for Thursday, and it seems overkill to list the sections of the page here.